genref/go.mod

```diff
@@ -0,0 +1,41 @@
+module github.ibm.com/crl-cloud-team/genref
+
+go 1.15
+
+require (
+	github.com/go-logr/logr v0.3.0 // indirect
+	github.com/kr/pretty v0.2.0 // indirect
+	github.com/pkg/errors v0.9.1
+	github.com/pmezard/go-difflib v1.0.0 // indirect
+	github.com/russross/blackfriday/v2 v2.0.1
+	github.com/shurcooL/sanitized_anchor_name v1.0.0 // indirect
+	github.com/tengqm/kubeconfig v0.0.0 // indirect
```

Comment:
What is the purpose of line 12 (kubeconfig)?

Reply:
It is a temporary hack because kubeadm config struct is not cleanly separated from the k/k repo while k/k is not meant to be directly imported by any other go software. This hack was suggested by sig-cluster-lifecycle. The vision is that we will eventually remove this when the kubeadm config is cleanly migrated into staging.